src/Focusable.ts

```diff
@@ -293,7 +294,7 @@ export class FocusableAPI implements Types.FocusableAPI {
             out.outOfDOMOrder = true;
         }
 
-        if (currentElement) {
+        if (currentElement && dom.nodeContains(container, currentElement)) {
```

Comment:
Does this address a bug that previously existed or was it added specifically for shadow roots?

Reply:
During the test I've discovered that native tree walker doesn't throw error if you set currentNode outside of the container. And my implementation had this constraint. So, I've just added the check as it seems to make sense.